src/_locales/ca/messages.json

```diff
@@ -596,23 +596,23 @@
         "description": "On-click text of the copy button on the share overlay."
     },
     "popup_special_page_header": {
-        "message": "Nothing to do on this page",
+        "message": "En aquesta pàgina no hi ha res a fer",
         "description": "Heading for popup_special_page_paragraph"
     },
     "popup_special_page_paragraph": {
-        "message": "Privacy Badger doesn't work on special pages like this one. Try browsing somewhere else.",
+        "message": "Privacy Badger no funciona en pàgines especals com aquesta. Proveu de navegar per altres llocs.",
         "description": "Shown in the popup for special browser pages such as the New Tab page and 'about:' pages."
     },
     "popup_disabled_site_header": {
-        "message": "Privacy Badger is disabled on this site",
+        "message": "En aquesta pàgina, el Privacy Badger està deshabilitat",
```

Comment:
We should use the translation for "this site", as opposed to "this page". (Search for "this site" elsewhere in the `en_US` messages file for related messages.)

Reply:
Yes, you're right. This is "En aquest lloc..." not "En aquesta pàgina..."